sqldelight-gradle-plugin/src/test/integration-postgresql/src/test/kotlin/app/cash/sqldelight/postgresql/integration/PostgreSqlTest.kt

```diff
@@ -186,4 +186,9 @@ class PostgreSqlTest {
       insertRef(RefTable(RefTable.Id(10), id))
     }
   }
+
+  @Test fun genRandomUuid() {
+    val uuid = database.datesQueries.randomUuid().executeAsOne()
```

Comment:
Could you add the type here to make sure it is really not null during compiling too?

Reply:
is this what you mean?
```suggestion
    val uuid : UUID = database.datesQueries.randomUuid().executeAsOne()
```

Reply:
Yes.